README.md

```diff
@@ -5,7 +5,8 @@
 ### Building:
 
 ```bash
-git clone https://github.com/opencontainers/runc
+git clone https://github.com/opencontainers/runc $GOPATH/src/github.com/opencontainers/runc
```

Comment:
Also as remove largest suffix prefix is POSIX defined http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_03_01 we can also use

```
${GOPATH%%:*}
```